platform/mbed_application.c

```diff
@@ -79,7 +79,7 @@ static void powerdown_nvic()
     int i;
     int j;
 
-#if defined(__CORTEX_M23)
+#if defined(__CORTEX_M23) || defined(__CORTEX_M33)
```

Comment:
Cortex-M33 does have an ICTR and can have up to 480 interrupts, so this change is not needed (and wrong)